cg/meta/archive/archive.py

```diff
@@ -12,6 +13,7 @@
 from pydantic import BaseModel, ConfigDict
 
 LOG = logging.getLogger(__name__)
+DEFAULT_SPRING_ARCHIVE_COUNT = 200
```

Comment:
What the actual default should be is up for debate. We will need to run a batch job first time archiving around 369K files. That will probably take a while. For reference, fetching all these files and linking them to a sample took 16 minutes.